builtin/credential/github/backend_test.go

```diff
@@ -26,7 +26,7 @@ func TestBackend_Config(t *testing.T) {
 
 	login_data := map[string]interface{}{
 		// This token has to be replaced with a working token for the test to work.
-		"token": "4fb5f4f0738c7aea5ee06dd595f15236ea78918b",
+		"token": os.Getenv("GITHUB_TOKEN"),
```

Comment:
I'm not a big fan of this change, we should use a restricted token that we just keep around. This requires that for the test to pass we set a token. Also, this change is weirdly unrelated to the PR.